### PR TITLE
Simplified Git Clone Instructions (Issue #2856)

### LINKED ISSUE #2856: Adding (Shift) + (Insert) to Paste GitHub Repo Link on Step 5
### Problem
On step 5, in the first step of "Cloning your GitHub Repository", the instructions tell the potential VI to type out the git clone command in this format: "git clone https://github.com/EmilyLarkin/EmilyLarkin.github.io.git". Although this method works, I think there is a better way to execute this. 

### Steps to reproduce the problem
I believe that the following is a better set of instructions:
1. Click the board icon next to the HTTPS link to copy the link to your clipboard
2. Open command prompt
3. Type "git clone"
4. Press (Shift)+(Insert) to paste the HTTPS link after git clone 
(CTRL + C does always for Command Line unless "QuickEdit Mode" is On)

### Screenshots
Git Pull 1

### Proposed solution
Revise this step to include the new proposed method, as well as revising the screenshot to highlight the copy to clipboard button next to the HTTPS link.

### PR DESCRIPTION
**Pull request by mattscodeans**
Fixes #2856 

The following pull request implements the simplified set of instructions for Git Clone detailed in Issue #2856. A new image file called "vi-github-clone-url-revised.jpg" is added in "vi/images", revising the old image by adding a blue marker around the clipboard button. The instructions under "Clone Your GitHub Repository username.github.io" have been revised to reflect the changes stated in Issue #2856.

https://raw.githack.com/mattscodeans/mattscodeans.github.io/simplified_git_clone/#!pages/vi/vi-github-and-repositories.md

### CODE REVIEW

File: pages/vi/vi-github-and-repositories.md [hunk 1]
```diff
@@ -30,11 +30,13 @@ This is just a summary of the steps that you will need to perform. Please, keep
...
 
-![GitHub Clone URL](images/vi-github-clone-url.png)
+![GitHub Clone URL](images/vi-github-clone-url-revised.jpg)
 
-Then, turn to your command prompt and type your repository URL in the form of `git clone https://github.com/EmilyLarkin/EmilyLarkin.github.io.git` into the command line. Be sure to use the correct URL to clone your repository (you will obviously type your own username).  If you  `cd  ` into your `username.github.io` you can see different files which you will be working on during your internship.
+Then, click on the green "Clone or Download" button and click the clipboard next to the HTTPS URL (highlighted in blue in image above). This will copy the link. Turn to your command prompt and type `git clone`. `git clone` is a git command line utility that makes a copy of an existing repository on your OS. Then, click (shift)+(insert) to paste your repository URL. Your command line should look like the following: `git clone https://github.com/Username/Username.github.io.git` (Username should be replaced with your own username).
+
+If you `cd` into your `username.github.io` you can see different files which you will be working on during your internship.
```
Comment (lmmrssa) [inline pages/vi/vi-github-and-repositories.md]: these are not files worked during internship. its just on 10 steps. internship will be different than this markdown

File: pages/vi/vi-github-and-repositories.md [hunk 2]
```diff
@@ -30,11 +30,13 @@ This is just a summary of the steps that you will need to perform. Please, keep
...
 
-![GitHub Clone URL](images/vi-github-clone-url.png)
+![GitHub Clone URL](images/vi-github-clone-url-revised.jpg)
 
-Then, turn to your command prompt and type your repository URL in the form of `git clone https://github.com/EmilyLarkin/EmilyLarkin.github.io.git` into the command line. Be sure to use the correct URL to clone your repository (you will obviously type your own username).  If you  `cd  ` into your `username.github.io` you can see different files which you will be working on during your internship.
+Then, click on the green "Clone or Download" button and click the clipboard next to the HTTPS URL (highlighted in blue in image above). This will copy the link. Turn to your command prompt and type `git clone`. `git clone` is a git command line utility that makes a copy of an existing repository on your OS. Then, click (shift)+(insert) to paste your repository URL. Your command line should look like the following: `git clone https://github.com/Username/Username.github.io.git` (Username should be replaced with your own username).
```
Comment (lmmrssa) [inline pages/vi/vi-github-and-repositories.md]: Shift insert will differ based on CLI used. Also it is not click. So this should not be added.

Comment (lmmrssa) [reply pages/vi/vi-github-and-repositories.md]: You should ask them to copy link of https but not guide in detail. As there are different ways to copy link. such as select and ctrl+C will do it or select right click and copy

File: pages/vi/vi-github-and-repositories.md (hunk 1 above)
Comment (mattscodeans) [reply pages/vi/vi-github-and-repositories.md]: Ok, I will take note. I believe that information was there before I made my changes.

File: pages/vi/vi-github-and-repositories.md (hunk 2 above)
Comment (mattscodeans) [reply pages/vi/vi-github-and-repositories.md]: Thank you so much, Immrssa, for reviewing my work. I have made the changes that you commented. I would like to say sorry for the delay. I had all my final exams last week and had to dedicate my time to my academics. Normally, I will produce changes at a much faster pace. May you please look at the changes I have made? Thank you in advance.

**Comment by mattscodeans** [comment]
Hello all, I have revised parts of the instructions. I made it so that copying the https link is not guiding the reader in full detail. I also removed the click (shift) + (insert) part due to it being different between CLIs. There was also an error with the last sentence, and I made changes to correct the misinformation.

**Pull request merged**